posnums.c

```diff
@@ -0,0 +1,91 @@
+#include <stdio.h>
+int main()
```

Comment:
Nice job overall! the only thing is that it doesn't print that 1 is prime when you input it as one of the numbers.